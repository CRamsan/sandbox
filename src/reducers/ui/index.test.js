import reducer from '.'

const initialState = {
  attentionBoxVisible: true,
}

describe('ui', () => {
  it('should return the initial state', () => {
    const next = reducer(initialState, {})
    expect(next).toEqual(initialState)
  })

  it('should return the initial state', () => {
    expect(
      reducer(initialState, { type: 'UI_ATTENTIONBOX_TOGGLE' })
    ).toEqual(
      {
        attentionBoxVisible: false,
      },
    )
  })

  it('UI_ATTENTIONBOX_TOGGLE', () => {
    const next = reducer(initialState, { type: 'UI_ATTENTIONBOX_TOGGLE' })
    expect(next).toMatchSnapshot()
  })
})
