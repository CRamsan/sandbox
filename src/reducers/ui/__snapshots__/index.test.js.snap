// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui UI_ATTENTIONBOX_TOGGLE 1`] = `
Object {
  "attentionBoxVisible": false,
}
`;
